pkg/kudoctl/cmd/init.go

```diff
@@ -149,6 +151,14 @@ func (initCmd *initCmd) run() error {
 			}
 			mans = append(mans, prereq...)
 
+			if opts.EnableValidation {
+				prereq, err := cmdInit.WebhookManifests(opts.Namespace)
```

Comment:
`WebhookManifests` likely needs to passed a options with a slice... or another parameter of slice of options.